csv_provider/README.md

```diff
@@ -42,3 +42,22 @@ Each line in the csv file consists of the following elements:
 | signal | the name of the signal to update | Vehicle.Speed
 | value | the new value | 48 |
 | delay | Indicates the time in seconds which the provider shall wait after processing this signal. This way one can emulate the change of signals over time. | 0 |
+
+## Recorder
+One way to generate a CSV-file for the CSV-provider is to record it from an Kuksa.val data broker. This way one can reproduce an interaction of different providers with the Kuksa.val data broker. The script in `csv_provider/recorder` allows this recording.
+An example call, only recording the vehicle speed and width would be:
+
+```
+pip install -r requirements.txt
+python3 provider.py
```

Comment:
Is is typo that you call provider.py here, and not recorder.py?

Reply:
True, thanks for pointing this out.